scripts/templateHelpers.ts

```diff
@@ -68,24 +66,6 @@ export function addNamespace(content: string, namespace: string): string {
   return processedTypes;
 }
 
-/**
- * Gets custom object id associated with the specified assetId.
- *
- * @param assetId - The assetId to look up
- *
- * @returns The custom object id as a string
- */
-export const getObjectIdByAssetId = (assetId: string): string => {
```

Comment:
We still need `getObjectIdByAssetId`, as the object IDs in the new help site are not generated from the RAML or metadata. The object IDs in the configuration need to be updated to match the new site.